qmd/languages.qmd

```diff
@@ -1,3 +1,77 @@
 # Languages {.unnumbered}
 
-The language table by langID.
\ No newline at end of file
+The language table by langID.
+
+
+```{r, echo=FALSE}
+# @RUSTY replace to use excel directly
+main_db <-
+  DBI::dbConnect(RSQLite::SQLite(), "../data/tables/main.sqlite")
+quary <-
+  paste("SELECT", "*", "FROM", "languages", "ORDER BY langID")
+languages_table <- DBI::dbGetQuery(main_db, quary)
+
+output_string <- ""
+
+# Loop over rows and create outputs per row
+for (row_index in 1:nrow(languages_table)) {
+  working_row <- languages_table[row_index,]
+  
+  nat_name <- working_row$natName
+  lang_ID <- working_row$langID
+  lang_fam <- working_row$langFam
+  lang_name <- working_row$langName
+  code_1 <- working_row$ISO6391
+  code_2 <- working_row$ISO6392B
+  code_3 <- working_row$ISO6392T
+  code_4 <- working_row$ISO6393
+  code_5 <- working_row$ISO6396
+  
+  # Check for not empty values
+  # @RUSTY consider functioning out checks for length
+  if (length(nat_name) < 1) {
+    nat_name <- "Native Name is missing at the momment"
```

Comment:
These should all be wrapped in functions and put next to the initialized variables